src/env.ts

```diff
@@ -28,4 +28,5 @@ export const env = {
   FEATURE_FLAGS: (process.env.FEATURE_FLAGS?.split(",") || []) as featureFlag[],
   WITHDRAWAL_FEE_MULTIPLIER: Number(process.env.WITHDRAWAL_FEE_MULTIPLIER || 4),
   HIRO_API_KEY: process.env.HIRO_API_KEY,
+  WITHDRAW_MIN_AMOUNT_SATS: Number(process.env.WITHDRAW_MIN_AMOUNT_SATS || 0),
```

Comment:
nit: we should default this to `547` instead of zero (unless we already have an explicit dust check somewhere else)

Reply:
I think we should assume this can only be undefined in development envs making it fun to play with